components/form-builder/app/responses/DownloadButton.tsx

```diff
@@ -46,6 +46,33 @@ export const DownloadButton = ({
       })
     );
 
+    // Download an aggregated version of all the records
+    if (checkedItems.size > 1) {
+      const urlAggregated = `/api/id/${formId}/submissions/download?format=html-aggregated`;
+      const ids = Array.from(checkedItems.keys());
+      axios({
+        url: urlAggregated,
+        method: "POST",
+        data: {
+          ids: ids.join(","),
+          // TODO temp for testing. need to determin how to handle languages
+          lang: "fr",
+        },
+      }).then((response) => {
+        const fileName = `${formId}.html`;
+        const href = window.URL.createObjectURL(new Blob([response.data]));
+        const anchorElement = document.createElement("a");
+        anchorElement.href = href;
+        anchorElement.download = fileName;
+        document.body.appendChild(anchorElement);
+        anchorElement.click();
+        document.body.removeChild(anchorElement);
+        window.URL.revokeObjectURL(href);
+      });
+
+      return; //TODO TEMP - when/where to download the aggregated file
```

Comment:
Can probably just leave off this wiring for now and just include the API response in this PR